.github/workflows/test-docs-mcp.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches: [main]
     paths:
       - "packages/mcp/**"
-      - "packages/docs-mcp/**"
+      - "packages/mcp-docs-server/**"
```

Comment:
Just noticed this path was also wrong so tests weren't running but I think tests still would've passed before since we weren't testing the bundled code